packages/cli/README.md

```diff
@@ -34,6 +36,16 @@ glaze COMMAND
 - [`glaze model:inspect NAME`](#glaze-modelinspect-name)
 - [`glaze model:list`](#glaze-modellist)
 - [`glaze model:publish NAME [OUTPUT]`](#glaze-modelpublish-name-output)
+- [`glaze pin:add STREAMID`](#glaze-pinadd-streamid)
+- [`glaze pin:ls [STREAMID]`](#glaze-pinls-streamid)
+- [`glaze pin:rm STREAMID`](#glaze-pinrm-streamid)
+- [`glaze stream:commits STREAMID`](#glaze-streamcommits-streamid)
+- [`glaze stream:state STREAMID`](#glaze-streamstate-streamid)
+- [`glaze tile:create CONTENT`](#glaze-tilecreate-content)
```

Comment:
I think we at least need an optional `--metadata` arg here.  Might want to make both metadata and content optional flags?

Reply:
The commands summary only prints the arguments, not the flags, but should be covered in the command doc.